log/logger_test.go

```diff
@@ -330,3 +330,94 @@ func TestRotateRightNow(t *testing.T) {
 		t.Fatalf("log rotate is not expected")
 	}
 }
+
+func TestRotateWithFormat(t *testing.T) {
+	err := InitGlobalRoller("format=mosn-2006-01-02_15.log")
+	if err != nil {
+		t.Fatal(err)
+	}
+	logName := "/tmp/mosn_bench/TestRotateWithFormat.log"
+	rollerName := "/tmp/mosn_bench/" + "mosn-" + time.Now().Format("2006-01-02_15") + ".log"
```

Comment:
after rotate, `TestRotateWithFormat.log` changed to another name `mosn-xxx`.
if there are two loggers or more, the log will be lose